nesta/core/batchables/eurito/cordis_eu/run.py

```diff
@@ -0,0 +1,134 @@
+    strans_kwargs={'filename':'eurito/cordis-eu.json',
+                   'from_key':'tier_0', 'to_key':'tier_1',
+                   'ignore':['id']}
+    es = ElasticsearchPlus(hosts=es_host,
+                           port=es_port,
+                           aws_auth_region=aws_auth_region,
+                           no_commit=("AWSBATCHTEST" in
+                                      os.environ),
+                           entity_type=entity_type,
+                           strans_kwargs=strans_kwargs,
+                           null_empty_str=True,
+                           coordinates_as_floats=True,
+                           listify_terms=True,
+                           do_sort=False,
+                           ngram_fields=['textBody_description_project'])
+
+    # collect file
+    logging.info('Retrieving project ids')
+    nrows = 20 if test else None
+    s3 = boto3.resource('s3')
+    obj = s3.Object(bucket, batch_file)
+    project_ids = json.loads(obj.get()['Body']._raw_stream.read())
+    logging.info(f"{len(project_ids)} project IDs "
+                 "retrieved from s3")
+
+    #
+    logging.info('Processing rows')
+    with db_session(engine) as session:
+        for count, obj in enumerate((session.query(Project)
+                                     .filter(Project.rcn.in_(project_ids))
+                                     .all())):
+            row = object_to_dict(obj)
+            
+            # Extract year from date
+            start_year = validate_date(row, 'start')
+            end_year = validate_date(row, 'end')
+            row['year'] = start_year if start_year is not None else end_year
+            
+            
+            _desc = row.pop('project_description')
+            _obj = row.pop('objective')
+            row['description'] = f'Description:\n{_desc}\n\nObjective:\n{_obj}'
+            row['link'] = f'https://cordis.europa.eu/project/id/{row["rcn"]}'
+
+            uid = row.pop('rcn')
+            _row = es.index(index=es_index, doc_type=es_type,
+                            id=uid, body=row)
+            if not count % 1000:
+                logging.info(f"{count} rows loaded to "
+                             "elasticsearch")
+
+    logging.warning("Batch job complete.")
+
+
+if __name__ == "__main__":
+    set_log_level()
+    if 'BATCHPAR_outinfo' not in os.environ:
+        from nesta.core.orms.orm_utils import setup_es
+        es, es_config = setup_es('dev', True, True,
+                                 dataset='cordis-eu')
+        environ = {'config': ('/home/ec2-user/nesta-cordis2es/nesta/'
+                              'core/config/mysqldb.config'),
+                   'batch_file' : ('cordis-eu_EURITO-ElasticsearchTask-2020-04-10-True-15865345336407135.json'),
+                   'db_name': 'dev',
+                   'bucket': 'nesta-production-intermediate',
+                   'done': "False",
+                   'outinfo': ('https://search-eurito-dev-'
```

Comment:
If we can expose this information here, why can't `elasticsearch.config` be a committed file?

Reply:
it's actually unnecessary for me hard code this - the info is already in `es_config`, so I've adapted accordingly

regarding why `elasticsearch.config` isn't public - it's kind of related to not making any of our db info public (obviously I screwed up in this case...)